Rubberduck.UnitTesting/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Rubberduck.UnitTesting")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Rubberduck.UnitTesting")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+[assembly: InternalsVisibleTo("RubberduckTests")]
+[assembly: InternalsVisibleTo("Rubberduck")]
+
+// Setting ComVisible to false makes the types in this assembly not visible
+// to COM components.  If you need to access a type in this assembly from
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("68f55404-32ca-4414-96b8-e284f3b1f846")]
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
```

Comment:
Should we match the `AssemblyFileVersion` (or `AssemblyVersion`?) to that of `Rubberduck.dll`?

Reply:
That needs be `Rubberduck.Deployment`'s job, I think, since it won't be just that assembly but all the other projects.

Reply:
Note that this issue is probably breaking EarlyBinding unit tests right now because the Code checking (and adding) the Rubberduck Reference to the test project assumes  they match.

Reply:
OK, but that's not the UnitTesting assembly's responsibility, is it? That's the responsibility of the assembly that does contains the types. Therefore, define a function to assert that the TLB is loaded correctly over there and have the UnitTesting assembly invoke it instead.

Reply:
so the EarlyBinding tests should be fixed with commit b23a1d7e9795f4256080af9542955e34fc29dfa6, I'll verify that with F5 testing tomorrow